File: CRM_View/Scripts/Employee.js
$(document).ready(function () {

    GetAllEmployee();

});

function GetAllEmployee()
{
    $.ajax({
        type: "GET", 
        contentType: "application/json; charset=utf-8",
        url: 'https://localhost:44329/api/Employees/GetEmployee',
        data: "",
        contentType: "application/json",
        success: OSGetAllEmployee,
        cache: false
    });
}
function OSGetAllEmployee(data,status) {

    var json = jsonParse(data.Result);
    $.each(json, function (i, option) {

        $('#GetAllData tbody').append(
            '<tr>' +
            
            '<td style="text-align:left">' + option.fullName + '</td>' +
            '<td style="text-align:left">' + option.lastName + '</td>' +
            '<td style="text-align:left">' + option.dateOfBirth + '</td>' +
            '<td style="text-align:left">' + option.email + '</td>' +
            '<td style="text-align:left">' + option.phoneNumber + '</td>' +
            '<td style="text-align:left">' + option.address + '</td>' +
            '<td style="text-align:left">' + option.city + '</td>' +
            '<td style="text-align:left">' + option.province + '</td>' +
            '<td style="text-align:left">' + option.zipCode + '</td>' +
            
            '</tr>'
        );
    });

}

File: API_CRM/Model/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API_CRM.Model
{
    public class Employee
    {
        public long EmployeeId { get; set; }
        public string FullName { get; set; }
        public string LastName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Email { get; set; }
        public long PhoneNumber { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string Province { get; set; }
        public long ZipCode { get; set; }
    }
}
